### ISSUE TITLE
Add content in Introduction to python

### ISSUE THREAD

**Issue by rohit0033** [thecuriousteam/PyLibLog#96]
![image](https://github.com/thecuriousteam/PyLibLog/assets/112507390/c9eb3981-ab75-4d24-b132-d1344fd34adc)
I can write the content for Python 

Can You please assign this issue to me

**Comment by thecuriousteam** [comment]
Yes you can start writing, Please join onboarding group for knowledge flow

Developers needs to onboard to opensource community for easy information flow along with admins, please join with this link  [Curious Developers Community - Hacktoberfest ](https://chat.whatsapp.com/HNu3VaLPK3z3yt3JFUs7D7)</br>

**Comment by Gyanendra-Verma** [comment]
i want to contribute to this issue , kindly assign this issue to me @thecuriousteam
